app/src/main/cpp/ControllerContainer.cpp

```diff
@@ -35,12 +35,14 @@ struct ControllerContainer::State {
   GeometryPtr beamModel;
   bool visible = false;
   vrb::Color pointerColor;
+  int gazeIndex;
```

Comment:
Same here, `int gazeIndex = -1`
Bellow it is initialized to `0` which I think not what we want?

Reply:
Fixed